eslint.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: eslint
+  version: 8.45.0
+  epoch: 0
+  description: An AST-based pattern checker for JavaScript
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - nodejs
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - nodejs
+      - rsync
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/eslint/eslint
+      tag: v${{package.version}}
+      expected-commit: 536cc343083ffd64172d4da13c7e043b140f1078
+
+  - runs: |
+      npm install --omit=dev
+      npm prune --production
+      ls -latr
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      install -Dm755 bin/${{package.name}}.js "${{targets.destdir}}"/usr/bin/${{package.name}}
+
+      rsync -r --exclude .cache --exclude .package-lock.json conf lib messages \
+          node_modules package.json "${{targets.destdir}}"/$_npmdir
```

Comment:
`$_npmdir` is not set here, so this will put the package straight into `/` in an image.